src/main/java/gov/usgs/wma/waterdata/data/DataController.java

```diff
@@ -59,14 +59,22 @@ public void getTimeSeries(
 					@ExampleObject(name = "waterML", value = "WaterML", description = "Water ML") }) @RequestParam(value = "f", required = false, defaultValue = "waterml") String mimeType,
 			HttpServletResponse response) throws HttpMediaTypeNotAcceptableException, IOException {
 
-		// not using the ContentType returned yet, since we only have waterML
-		determineContentType(mimeType, List.of(ContentType.waterml));
+		ContentType contentType = determineContentType(mimeType, List.of(ContentType.json, ContentType.waterml));
 		String rtn = null;
 		String bestTS = best == null ? CollectionParams.PARAM_MATCH_ANY : best.toString().toLowerCase();
 
+		if (contentType.isJson() && !bestTS.equals("true")) {
+			throw new HttpMediaTypeNotAcceptableException("Json content is only available with parameter best=true");
```

Comment:
Why is this condition part of the contract?

Reply:
Currently, the json (omsf) is not row based, it only has room for one observed_property (pcode) in the properties object.
Even if the extra pcode was stored overloaded in the properties, there would be no way to map them to the one result array. Hence, limiting to one time series (best=true),

Reply:
good question.  This is certainly worth of a comment. I'll add some notes.